File: SOURCE_DOCSUITE/VecompSoftware.DocSuiteWeb.Gui/App/Models/Processes/ProcessFascicleTemplateModel.ts
import ProcessModel = require("App/Models/Processes/ProcessModel");
import DossierFolderModel = require("App/Models/Dossiers/DossierFolderModel");

interface ProcessFascicleTemplateModel {
    UniqueId: string;
    Process: ProcessModel;
    DossierFolder: DossierFolderModel;
    Name: string;
    JsonModel: string;
    StartDate: Date;
    EndDate: Date;
    RegistrationUser: string;
    RegistrationDate: Date;
    LastChangedUser: string;
    LastChangedDate: Date;
}

export = ProcessFascicleTemplateModel;

File: SOURCE_WEBAPI/VecompSoftware.DocSuiteWeb.Model/Entities/Processes/ProcessFascicleTemplateModel.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace VecompSoftware.DocSuiteWeb.Model.Processes
{
    public class ProcessFascicleTemplateModel
    {
        #region [ Constructors ]

        public ProcessFascicleTemplateModel()
        {
        }

        #endregion

        #region[ Properties ]

        public Guid? UniqueId { get; set; }
        public string Name { get; set; }
        public string JsonModel { get; set; }
        public DateTimeOffset StartDate { get; set; }
        public DateTimeOffset? EndDate { get; set; }
        public DateTimeOffset RegistrationDate { get; set; }
        public string RegistrationUser { get; set; }

        #endregion

    }
}
